src/plugins/intel_gna/gna_plugin.cpp

```diff
@@ -1597,15 +1598,25 @@ InferenceEngine::IExecutableNetworkInternal::Ptr GNAPlugin::ImportNetwork(std::i
     GNAModelSerial::MemoryType mt;
     auto serial = GNAModelSerial(&model->object(), mt);
 
-
     serial.setHeader(header);
     serial.Import(basePtr,
                   header.gnaMemSize,
                   networkModel,
                   *(inputs_ptr_),
                   outputs_,
                   transpose_inputs_info,
-                  transpose_outputs_info);
+                  transpose_outputs_info,
+                  modelLibVersion);
+
+    // Print OV and GNA Lib versions used for model export
+    if (gnaFlags->log_level == ov::log::Level::DEBUG) {
```

Comment:
The same condition is already checked in 
https://github.com/marcinkusm/openvino/blob/1afd8460e3128ed34d810a70de5e88083188b091/src/plugins/intel_gna/gna_plugin.cpp#L381

If it is possible maybe it would be good create method which return true in case log is debug, otherwise false.
It could be used in two places

Reply:
I would disagree, this will make code more complicated, this is pretty clear condition now.
But we are incorrectly using ov::log::Level::Debug, this is level of logging and we have to print this info for TRACE level too. Please correct the condition

Reply:
```suggestion
    if (gnaFlags->log_level >= ov::log::Level::DEBUG) {
```

Reply:
The best way to move the logic should it be trace or debug or just info log to separate function, eg gna_log(ov::log::Level::DEBUG) and there we can compare current plugin log level and decide print or not this log.

Reply:
fiexed